src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const obj = {};
```

Comment:
[Checklist](https://github.com/mate-academy/js_inverse_robot/blob/master/checklist.md) 3rd item, use more appropriate names for variables, smth like `inversedRobot` would be a great fit